### PR TITLE
Wang Leng: Update resume and expert areas

### PR DESCRIPTION
**Pull request by yamgent**

### CODE REVIEW

File: students/AY1617S2/tanWangLeng/TanWangLeng-Resume.md [hunk 1]
```diff
@@ -33,15 +45,58 @@ Information Systems Intern
...
+## Project Experiences
+
+### Contributions to OSS Projects
+
+External Projects:
+* [GitHub's Electron](https://github.com/electron/electron) (Contributor)
+    * [Fix webpreferences not accepting numeric options #8542](https://github.com/electron/electron/pull/8542)
+    * [Support message, nameFieldLabel and showsTagField for macOS dialog #8556](https://github.com/electron/electron/pull/8556)
+
+Internal Projects (for CS3281&2: Thematic Systems Project I/II):
```
Comment (damithc) [inline students/AY1617S2/tanWangLeng/TanWangLeng-Resume.md]: Just list the project in the order of importance. No need to mention external/internal or module codes.

File: students/AY1617S2/tanWangLeng/TanWangLeng-Resume.md [hunk 2]
```diff
@@ -33,15 +45,58 @@ Information Systems Intern
...
+
+* CP3108B: Independent Work [Orbital]
+    * Archive Repository: [Link](https://github.com/yamgent/orbital-2016)
+    * Used the Ruby on Rails framework
+    * Wrote a prototype website to explore possible improvements that could be made for the interface of the NUS course bidding website used by undergraduates.
+
+
+* Major Project (Temasek Polytechnic)
+    * As a lead programmer, developed a top-down 3D game in Unity3D engine titled "Morality" with a team of 5 other fellow student developers.
+    * Targeted towards youths aged 10 - 16, with the aim of teaching them good moral values and habits.
```
Comment (damithc) [inline students/AY1617S2/tanWangLeng/TanWangLeng-Resume.md:76]: Not required for this module, but you may want to create a webpage with some screenshots/videos/details of the game and link from here.

**Review by damithc — COMMENTED** [review]
Added some comments in passing. Let me know when ready to merge (up to you).

**Comment by yamgent** [comment]
Thank you for your suggestions! :D

Added in improvements. This iteration of the resume is ready for merging.

**Pull request merged**